### ISSUE TITLE
OutOfRangeError 

### ISSUE THREAD

**Issue by kokhazad** [yscacaca/DeepSense#35]
Hi @yscacaca ,

Thank you for sharing your source code.
I tried to run your source code but faced the below error:

`OutOfRangeError (see above for traceback): RandomShuffleQueue '_2_shuffle_batch/random_shuffle_queue' is closed and has insufficient elements (requested 64, current size 14`

I tried to change tensorflow version 10 times between 1.1 to 1.10 with python 2.7 and it didn't solve the problem.
I've read the previous issues on this problem and didn't find a proper solution.
Is it possible to share the packages versions here? like python, tensorflow, ...
Or share all requirement packages versions as a yml file?

It gives me the error on the below line:
`
example_batch, label_batch = tf.train.shuffle_batch(
	[example, label], batch_size=batch_size, num_threads=16, capacity=capacity,
	min_after_dequeue=min_after_dequeue)
`
